README.md

```diff
@@ -0,0 +1,55 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
+
+My first Hello Microverse repo
+
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML & CSS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+clone repo: `git clone https://github.com/butlermuwo/Hello-Microverse`
+
+then
+`cd My Hello Microverse project`
+
+### Install
+
+run `npm install` to install dependencies
+
+## Authors
+
+👤 **Author**
+
+- GitHub: [@butlermuwo](https://github.com/butlermuwo)
+- Twitter: [@ButlerMuwo](https://twitter.com/ButlerMuwo)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/butler-shimaluwani-41a680159)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
The link for the license on your repository appears to be broken. 

![image](https://user-images.githubusercontent.com/77038610/143055522-722fc19e-b273-4784-b480-9454297f1e1d.png)

- [ ] Kindly fix the license link. You have deleted  a file called `license`, you could restore it and rename it to MIT.md. Having functional links on the README is very important to give a professional appearance to your repository 👍🏻